taxi/forms.py

```diff
@@ -0,0 +1,57 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+def validation_license_number(license_number):
+    length = 8
+    first_char = 3
+    last_char = 5
+
+    if len(license_number) != length:
+        raise ValidationError(f"Must consist only of {length} characters")
+    if not license_number[:first_char].isalpha() or \
+            not license_number[:first_char].isupper():

```

Comment:
Prefer not to use `\` for line breaking, place the whole expression inside of `()`.